distributed/scheduler.py

```diff
@@ -2086,6 +2091,16 @@ def _transition(
             return recommendations, client_msgs, worker_msgs
         except Exception:
             logger.exception("Error transitioning %r from %r to %r", key, start, finish)
+            self.log_event(
+                "transistions",
+                {
+                    "action": "transition-failure",
```

Comment:
nit: Once again, no clear guideline exists, but I'd personally prefer (and it seems somewhat more commonly used) to include a verb in the action's name. Unless this is moved to a `scheduler` topic, I'd also prefix this with `scheduler-` to ensure this is distinct from any future worker-based logs regarding transaction failures.
```suggestion
                    "action": "scheduler-transition-failed",
```